vtex-apps/product-summary
# Feat/mediaquery opacity mobile

#### What problem is this solving?

Hover effect into product summary for small devices is vanishing images in mobile duo this:

 https://github.com/vtex-apps/product-summary/blob/2a39af1631d8f55fb22884a0c25f080d42278bc0/react/productSummary.css#L90-L92


<img src="https://user-images.githubusercontent.com/13649073/116487935-8caa4900-a867-11eb-8b57-8d86a36bf1b7.png" width=400>

#### How to test it?

Enter in the workspace and check pressing the image when in mobile device

[Workspace](https://hoverimage--lamparayamx.myvtex.com/)

#### Screenshots or example usage:

<img src="https://user-images.githubusercontent.com/13649073/116488037-c8451300-a867-11eb-816d-5bd0fbd12549.png" width=400>




#### Related to / Depends on

https://github.com/vtex-apps/product-summary/issues/304
https://github.com/vtex-apps/product-summary/pull/283

#### How does this PR make you feel? [:link:](http://giphy.com/)

![](https://media.giphy.com/media/3orifbkk22REj32o9O/giphy.gif)
